jupyterlab_caip_optimizer/src/service/test-contsants.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * For easier testing of code using nested objects
+ */
+import {
+  Study,
+  StudyConfig,
+  MetricSpec,
+  ParameterSpec,
+  ParameterType,
+  Algorithm,
+  GoalType,
+  State,
+} from '../types';
+
+export const fakeMetric1 = {
```

Comment:
I think for all these constants instead of using numbers I would distinguish them further (i.e. fakeMetricUnspecified, fakeParamCategorical, fakeStudyConfigUnspecified) so it's easier to tell what they do